opus/application/static_media/js/opus.js

```diff
@@ -196,7 +196,7 @@ var opus = {
         $("#op-result-count").html(opus.spinner).parent().effect("highlight", {}, 500);
 
         // Start the observation number slider spinner - no point in doing a flash here
-        $("#browse .op-observation-number").html(opus.spinner);
+        $(".op-observation-number").html(opus.spinner);
```

Comment:
Are you sure you want to do this for both sliders? Just because the search has changed doesn't mean we need a spinner on the cart. They are unrelated.

Reply:
good catch, updating to just set the spinner on the #browse tab.